UnitsNet.Tests/QuantityTests.ToString.cs

```diff
@@ -124,9 +124,9 @@ public void FormatsNumberUsingGivenDigitsAfterRadix()
                 try
                 {
                     GlobalConfiguration.DefaultCulture = CultureInfo.InvariantCulture;
-                    Assert.Equal("0.05 m", Length.FromCentimeters(5).ToUnit(LengthUnit.Meter).ToString(null, 4));
-                    Assert.Equal("1.97 in", Length.FromCentimeters(5).ToUnit(LengthUnit.Inch).ToString(null, 2));
-                    Assert.Equal("1.9685 in", Length.FromCentimeters(5).ToUnit(LengthUnit.Inch).ToString(null, 4));
+                    Assert.Equal("0.05 m", Length.FromCentimeters(5).ToUnit(LengthUnit.Meter).ToString("s4"));
+                    Assert.Equal("1.97 in", Length.FromCentimeters(5).ToUnit(LengthUnit.Inch).ToString("s2"));
```

Comment:
Maybe construct the meter and inch quantities directly in that unit, so it's easier to compare the input and the output numeric values - instead of using `ToUnit()`.

Reply:
Not sure I follow.

I just left the code as-is and replaced the format string due to the obsolete warning causing a build failure.

Reply:
Ah yes, sorry, what I commented on was probably not a change introduced in your PR but rather something I just noticed while reviewing and maybe something we could fix while at it.

What I was trying to convey was that, instead of this
```c#
Assert.Equal("1.97 in", Length.FromCentimeters(5).ToUnit(LengthUnit.Inch).ToString("s2"));
```
Maybe do this:
```c#
Assert.Equal("1.2345 in", Length.FromInches(1.2345).ToString("s2"));
```

Reply:
I agree! However I would have to do all the conversions to make sure the double has enough decimal places on construction to make sure it's testing the same thing. I'd rather not do that as part of this PR!

Reply:
Fair enough